.github/workflows/tests.yml

```diff
@@ -32,3 +34,22 @@ jobs:
 
         # Test tools
         python3 -m unittest discover -vs tools/tests -p 'test_*.py'
+
+  test-github-scripts:
+    name: Test github scripts
```

Comment:
BTW, where and when is this job running?

Reply:
This is the test job for the fetch latest green commit script, so it will run whenever the script is updated and on test-infra, i.e. https://github.com/pytorch/test-infra/actions/runs/7634750369/job/20799105964?pr=4905#step:11:197